### PR TITLE
[fltk] Force static library due to global variables. Fix CMake Targets.

### PR DESCRIPTION
**Pull request by ras0219**
See title.

### CODE REVIEW

File: ports/fltk/CONTROL [hunk 1]
```diff
@@ -1,5 +1,5 @@
 Source: fltk
-Version: 1.3.5-2
+Version: 1.3.5-3
```
Comment (strega-nil) [inline ports/fltk/CONTROL:2]:
```suggestion
Version: 1.3.5
Port-Version: 3
```

**Review by strega-nil — CHANGES_REQUESTED** [review]
That's the only thing I see :)

**Comment by PhoebeHui** [comment]
The bin folder need to remove.

Failures on OSX:

-- Performing post-build validation
There should be no bin\ directory in a static build, but /Users/vagrant/Data/packages/fltk_x64-osx/bin is present.
There should be no debug\bin\ directory in a static build, but /Users/vagrant/Data/packages/fltk_x64-osx/debug/bin is present.
If the creation of bin\ and/or debug\bin\ cannot be disabled, use this in the portfile to remove them

    if(VCPKG_LIBRARY_LINKAGE STREQUAL "static")
        file(REMOVE_RECURSE "${CURRENT_PACKAGES_DIR}/bin" "${CURRENT_PACKAGES_DIR}/debug/bin")
    endif()

Found 1 error(s). Please correct the portfile:
    /Volumes/data/work/1/s/ports/fltk/portfile.cmake
-- Performing post-build validation done
Error: Building package fltk:x64-osx failed with: POST_BUILD_CHECKS_FAILED
Elapsed time for package fltk:x64-osx: 1.072 min

File: ports/fltk/portfile.cmake [hunk 2]
```diff
@@ -34,56 +33,31 @@ vcpkg_configure_cmake(
...
+    vcpkg_copy_tools(TOOL_NAMES fluid.app fltk-config AUTO_CLEAN)
+elseif(VCPKG_TARGET_IS_WINDOWS)
+    file(REMOVE ${CURRENT_PACKAGES_DIR}/bin/fltk-config ${CURRENT_PACKAGES_DIR}/debug/bin/fltk-config)
+    vcpkg_copy_tools(TOOL_NAMES fluid AUTO_CLEAN)
+else()
+    vcpkg_copy_tools(TOOL_NAMES fluid fltk-config AUTO_CLEAN)
 endif()
 
+vcpkg_copy_pdbs()
+
```
Comment (PhoebeHui) [inline ports/fltk/portfile.cmake:55]:
```suggestion

if(VCPKG_LIBRARY_LINKAGE STREQUAL static)
    file(REMOVE_RECURSE
        ${CURRENT_PACKAGES_DIR}/debug/bin
        ${CURRENT_PACKAGES_DIR}/bin
    )
endif()
```

**Review by PhoebeHui — APPROVED** [review]

**Review by LilyWangL — APPROVED** [review]

**Review by strega-nil — APPROVED** [review]

**Pull request merged**